specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/webapplicationfirewall.json

```diff
@@ -453,13 +453,28 @@
           "type": "boolean",
           "description": "Whether to allow WAF to check request Body."
         },
+        "reqBodyInspectLimitInKB": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 8,
+          "exclusiveMinimum": false,
+          "description": "Max inspection limit in KB for request body inspection for WAF."
+        },
+        "requestBodyEnforcement": {
+          "type": "boolean",
+          "description": "Whether allow WAF to enforce request Body limits."
```

Comment:
"B" is capitalized in "Body" in this line, but nowhere else. This should probably be lowercase.